output/json.go

```diff
@@ -0,0 +1,75 @@
+package output
+
+import (
+	"encoding/json"
+	"github.com/yarpc/crossdock/execute"
+	"github.com/yarpc/crossdock/plan"
+	"io/ioutil"
```

Comment:
[Import group ordering](https://github.com/golang/go/wiki/CodeReviewComments#imports)

You want,

```go
import (
  // standard library

  // local packages

  // third party packages
)
```

`goimports` will do part of this for you if you have it set up.